packages/neovim/src/utils/findNvim.ts

```diff
@@ -137,9 +189,8 @@ export function findNvim(opt: FindNvimOptions = {}): Readonly<FindNvimResult> {
         const luaJitVersionMatch = luaJitVersionRegex.exec(nvimVersionFull);
         if (nvimVersionMatch && buildTypeMatch && luaJitVersionMatch) {
           if (
-            'minVersion' in opt &&
-            compareVersions(opt.minVersion ?? '0.0.0', nvimVersionMatch[1]) ===
-              1
+            opt.minVersion &&
+            compareVersions(opt.minVersion, nvimVersionMatch[1]) === 1
```

Comment:
the old logic supported callers that explicitly pass `minVersion = undefined`.

Reply:
Oops, my bad